src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,9 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
Yes I think so. The only time I think it should work is when double clicking a favourite.